src/common/lib/client/auth.ts

```diff
@@ -618,41 +663,34 @@ class Auth {
           );
           return;
         }
-        if ('issued' in tokenRequestOrDetails) {
+        // Not sure why we need the ! in the next two uses of tokenRequestOrDetails, given the preceding `typeof === 'object'` check
```

Comment:
> `typeof null === 'object'`

😱 (didn't know this)

> I think it's a bug and suggest changing (…)

Will keep the `object` check since I guess it's also meant to be checking that somebody hasn't accidentally passed a number or something, and will add a `== null` check.

Reply:
> Will keep the object check since I guess it's also meant to be checking that somebody hasn't accidentally passed a number or something

I didn't think about that and I agree, can check for that too.